packages/ui/core-components/src/lib/unsorted/viz/map/components/Points.svelte

```diff
@@ -287,7 +291,9 @@
 		<Point
 			{map}
 			options={{
-				fillColor: color ?? (value ? colorScale(item[value]).hex() : uiColours.blue700), // Fill color of the circle
+				// kw note:
+				//need to clean this logic
+				fillColor: color ?? map.handleFillColor(item, value, values, colorPalette, colorScale),
```

Comment:
Any changes needed here?

Reply:
nope, it was cleaned, thanks! will remove comment